sqlglot/expressions.py

```diff
@@ -90,7 +90,7 @@ def set(self, arg, value):
             value: value to set the arg to.
         """
         self.args[arg] = value
-        self._set_parent(arg=value)
+        self._set_parent(**{arg: value})
```

Comment:
what’s up with this one?

Reply:
This is fixing a bug.

self._set_parent(arg=value) is different than self._set_parent(**{arg: value})

Reply:
oh wow it is